redhat/pimd.spec

```diff
@@ -45,7 +46,7 @@ make %{?_smp_mflags}
 
 %files
 %defattr(-,root,root,-)
-%doc AUTHORS CREDITS FAQ INSTALL LICENSE LICENSE.mrouted README README.config README.config.jp README.debug RELEASE.NOTES TODO
+%doc AUTHORS CREDITS LICENSE LICENSE.mrouted RELEASE.NOTES
```

Comment:
You can drop CREDITS and RELEASE.NOTES as well, and include the ChangeLog PDF and/or .org file instead.